spec/controllers/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe API::V1::KeywordsController, type: :controller do
+  describe 'POST#create', api_authentication: :user do
+    context 'when upload a valid file' do
+      it 'returns created status' do
+        sign_in Fabricate(:user)
+        post :create, params: keywords_file_params('valid_keywords.csv')
+
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'when upload no valid file' do
```

Comment:
Can we test with the whole response also(not just the status)?

Reply:
Fixed in [`65180c8` (#54)](https://github.com/suho/google-search-ruby/pull/54/commits/65180c8d4761f3f08843a6c32f6b92e66f132383)